app/models/financial_aid/my_awards.rb

```diff
@@ -209,18 +209,33 @@ def sub_items(award_type, item)
           alertDetails: alert_details(item['item_type']),
           termDetails: disbursements(item['item_type']),
           awardMessage: item['award_message'],
+          authFailedMessage: auth_failed_message(item['item_type']),
           itemTypeDescr: item['title']
         }
       else
         {
           alertDetails: alert_details(item['item_type']),
           remainingAmount: (item['left_col_amt'] - item['right_col_amt']).to_f,
           awardMessage: item['award_message'],
+          authFailedMessage: auth_failed_message(item['item_type']),
           itemTypeDescr: item['title']
-        } 
+        }
       end
     end
 
+    def auth_failed_message_exists?(item_type)
+      exists ||= EdoOracle::FinancialAid::Queries.get_auth_failed_message(@uid, my_aid_year, item_type)
+      !!exists
+    end
+
+    def auth_failed_message(item_type)
+
+      return nil unless auth_failed_message_exists?(item_type)
+
+      auth_message ||= CampusSolutions::MessageCatalog.get_message(:financial_aid_awards_card_auth_failed).try(:[], :descrlong)
```

Comment:
@marcpechaitis - You're not using an instance variable here either, and the result of this method depends on `item_type` so you might want to initialize a hash and use it to store the result of those external calls, just like I suggested above.